app/assets/stylesheets/bali/forms.scss

```diff
@@ -0,0 +1,112 @@
+.input.is-very-short {
+  width: 4rem;
+}
+
+.input.is-short {
+  width: 100px;
+}
+
+.input.is-medium-short {
+  width: 140px;
+}
+
+.input.is-long {
+  width: 16rem;
+}
+
+.select.full-width {
+  width: 100%;
+
+  select,
+  .select2-container {
+    width: 100% !important;
+  }
+}
+
+.control.is-small {
+  width: 10rem;
+}
+
+.is-danger .select2-container .select2-selection--multiple {
+  border-color: $danger;
+}
```

Comment:
I think select2 was used by a previous select library we were using, please validate and delete if it's not used anymore